### DISCUSSION TITLE
Global vs profile settings

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#319] [Q&A] [answered]
My second minor suggestion is to indicate whether a setting is global or just for a profile (like car, bike, running…) In the moment I do not know whether a color setting or something else is a global setting or not. I propose to indicate or to separate the settings in global and profile settings.

**Comment by devemux86** [comment]
@kds112 thanks for the suggestion.

It's very simple, all app settings are specific to each profile, except:
- App language
- Storage folder

**Reply by kds112** [reply]
Oh, that is easy! Thanks! Even the selected maps, themes and theme settings? E. g. the elevate theme has different layers for hiking, biking with selectable attributes. Are these setting stored for each profile ?

**Reply by devemux86** [reply]
Active maps / poi are currently global, as it is a bit complicated to handle them separately for each profile.

e.g. we delete a map / theme / poi in one profile and then activate another profile that wants the deleted data

**Reply by kds112** [reply]
Well, for the Elevate theme it is not the best solution. When I change the profile from Hiking to Biking I have to change the theme layer and the layer attributes in a second and third step because the selected/unselected attributes of the theme layer are not stored...

**Reply by devemux86** [reply]
They are always stored, the last selected ones as they are global.

I might look at it in the future, it requires a careful implementation.

**Reply by kds112** [reply]
I am sorry but the selection of the theme layer attributes is not stored for the Elevate theme. E.g. I am using the Hiking Layer and change the selection of the attributes. Then I change the layer from Hiking to Biking. After that I change back to the Hiking layer. The selection of the attributes returns to the default selection. My last selection in the Hiking layer is not remembered. Changing the layer always sets the attribute selection to a default selection. Can that be modified?

**Reply by devemux86** [reply]
It seems you mean something different and not related to the profiles.

When we are in the theme layers screen, we cannot remember every combination of selections.
The last active selections are the ones that are stored.

It could probably work if we connect the map / themes with the profiles, as suggested above.

**Comment by devemux86** [comment]


**Reply by devemux86** [reply]
Please wait for the next Beta version.

**Comment by devemux86** [comment]
Profiles now also restore the map theme + layers. You can test it in latest:
- https://github.com/devemux86/cruiser/discussions/329

**Reply by kds112** [reply]
It works as expected. Thanks a lot! But now I can't sort the profiles with drag&drop anymore. Can you check it?

**Reply by devemux86** [reply]
@kds112 drag them from the left icon (without long press).

**Reply by kds112** [reply]
Ahhh... Okay... Didn't know it. Thanks :-)

**Reply by devemux86** [reply]
We changed it as it was before for easier drag to reorder. Please see also the discussion here:
- https://github.com/devemux86/cruiser/discussions/314

**Reply by DolomitesTom** [reply]
Could we also (additionally) make it dragable when tapping on the text (complete frame)? Because then it could be used easier with the right thumb for right-handers.

**Reply by devemux86** [reply]
@DolomitesTom please see here:
- https://github.com/devemux86/cruiser/discussions/314#discussioncomment-7146235

**Comment by devemux86** [comment, marked as answer]
> Profiles now also restore the map theme + layers.

Available in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
